### PR TITLE
Document home manager plugins array

### PR DESCRIPTION
**Pull request by outfoxxed**
Companion pr for hyprwm/Hyprland#2211

### CODE REVIEW

**Comment by viperML** [comment]
This doesn't explain how to get the plugin packages

File: pages/Configuring/Dispatchers.md [hunk 1]
```diff
@@ -97,8 +97,6 @@ You have eight choices:
 
 - Relative workspace on monitor: e.g. `m+1`, `m-1` or `m+3`
 
-- Relative workspace on monitor including empty workspaces: e.g. `r+1` or `r-3`
```
Comment (fufexan) [inline pages/Configuring/Dispatchers.md:100]: Why remove this? I don't see how it's related to this PR.

File: pages/Nix/Options & Overrides.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ programs.hyprland = { # or wayland.windowManager.hyprland
 };
 ```
 
+### HM module
```
Comment (fufexan) [inline pages/Nix/Options & Overrides.md:32]: This should go below `## Options descriptions`, before or after `### XWayland`. It should be something like

    ### Plugins

    ```nix
    wayland.windowManager.hyprland = {
      plugins = [
        inputs.hyprland-plugins.packages.${pkgs.system}.hyprbars # package
        "/home/<user>/Projects/my-hyprland-plugin/libmy-hyprland-plugin.so" # path
      ];
    };
    ```

And then you should include more info about how to build plugins, like "look at how pacakges are defined in `hyprland-packages`".

**Review by fufexan — CHANGES_REQUESTED** [review]

File: pages/Configuring/Dispatchers.md (hunk 1 above)
Comment (outfoxxed) [reply pages/Configuring/Dispatchers.md:100]: No clue how that got in here. Good catch.

File: pages/Nix/Options & Overrides.md (hunk 2 above)
Comment (outfoxxed) [reply pages/Nix/Options & Overrides.md:32]: Alright

Comment (outfoxxed) [reply pages/Nix/Options & Overrides.md]: Done. You probably want to squash this.

**Comment by fufexan** [comment]
Okay, it looks good enough for now. The Nix part of the wiki will need a big rewrite anyway once I can get a plugin builder in Nix.

**Review by viperML — APPROVED** [review]

**Pull request merged**